assets/src/components/home/Home.tsx

```diff
@@ -28,6 +29,7 @@ export default function Home() {
       />
       <HomeContentWrapperSC>
         <ClusterOverviewCard />
+        {me?.roles?.admin && <ConstraintViolationsCard />}
```

Comment:
This isn't the field we'd make it conditional on.  It'd actually be from the users personas and if one of them specifies the `home` field as `manager`.

Reply:
Ah yeah that makes more sense. fixed (made a test persona to confirm all seems to work now)